src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -41,6 +41,7 @@
 import org.apache.logging.log4j.Logger;

```

Comment:
In this file, you may be also able to remove cases for `throwableprojectile` and similar that are no longer used.